### PR TITLE
add admin auth check for api /configuration/createSecondCategory and /microservice/allInstance

### PR DESCRIPTION
**Pull request by legendtkl**
### What is the purpose of the change
ref issue: https://github.com/apache/incubator-linkis/issues/1867

### Brief change log
- add admin check for api

### Verifying this change
This change added tests and can be verified as follows:  
- mannually request api

### Does this pull request potentially affect one of the following parts:
- Dependencies (does it add or upgrade a dependency): ( no)
- Anything that affects deployment: ( no )
- The MGS(Microservice Governance Services), i.e., Spring Cloud Gateway, OpenFeign, Eureka.: ( no)

### Documentation
- Does this pull request introduce a new feature? ( no)
- If yes, how is the feature documented? (not applicable )

### CODE REVIEW

**Comment by legendtkl** [comment]
Hi, @peacewong , would you help review this PR?

Thanks,
Kelu.

File: linkis-public-enhancements/linkis-publicservice/linkis-instance-label/linkis-instance-label-server/src/main/java/org/apache/linkis/instance/label/restful/InstanceRestful.java [hunk 1]
```diff
@@ -126,4 +136,9 @@ public Message getEurekaURL(HttpServletRequest request) throws Exception {
         String eurekaURL = insLabelService.getEurekaURL();
         return Message.ok().data("url", eurekaURL);
     }
+
+    private Boolean isAdmin(String userName) {
+        String[] admins = InstanceConfigration.GOVERNANCE_STATION_ADMIN().getValue().split(",");
```
Comment (peacewong) [inline linkis-public-enhancements/linkis-publicservice/linkis-instance-label/linkis-instance-label-server/src/main/java/org/apache/linkis/instance/label/restful/InstanceRestful.java:141]: It is recommended to optimize direct calls: Configuration.isAdmin

**Review by peacewong — APPROVED** [review]
LGTM.

**Review by peacewong — APPROVED** [review]
LGTM.

**Pull request merged**